### ISSUE TITLE
Recent "This venue is not set up to accept payment methods" Issues

### ISSUE THREAD

**Issue by blakejkaplan** [Alkaar/resy-booking-bot#101]
I've recently started getting the following issue for some restaurants requiring payment when creating a reservation:

```
[DEBUG] 2023-03-26 09:00:08.137-04:00 ResyClient:119 - URL Response: {"status": 403, "code": null, "message": "This venue is not set up to accept payment methods. Please contact the venue for assistance.", "data": {}}
```

As far as I can tell, the payment ID and booking token are being correctly determined but the booking request is getting rejected with a 403. I think this only started happening recently. Let me know what you think.

**Comment by Alkaar** [comment]
Just double-checking you also looked at my note here as Resy made a change on their end regarding how they manage payments 👉 https://github.com/Alkaar/resy-booking-bot/discussions/92

If you could share the restaurant where this is happening, I can test it for myself to see if I get the same error. Ideally, a restaurant that has more readily available reservations. I was able to use the bot today and I did not run into this issue.

**Comment by blakejkaplan** [comment]
I definitely missed #92 so will give that a look. The restaurant was 4 Charles so not exactly readily accessible reservations

**Comment by blakejkaplan** [comment]
I think this is fixed but now I'm seeing #102

I'll close this out since it's being tracked separately.
